swiftui/PasswordlessSwiftUI/Podfile

```diff
@@ -0,0 +1,21 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '14.0'
```

Comment:
Is 13.0 possible?

Reply:
14 is required for the new `@main` and entire-SwiftUI  lifecycle (instead of AppDelegate)